kolibri/plugins/management/assets/src/state/store.js

```diff
@@ -41,7 +41,10 @@ const mutations = {
   SET_PAGE_STATE(state, pageState) {
     Object.assign(state.pageState, pageState);
   },
-
+  SET_CONTENT_PAGE_STATE(state, pageState) {
+    state.pageName = 'CONTENT_MGMT_PAGE';
+    state.pageState = pageState;
+  },
```

Comment:
Is this PR ready?

I see `SET_CONTENT_PAGE_STATE` added, but no references to it; also `SET_PAGE_STATE` still uses `Object.assign`

Reply:
Ugh, it committed precisely the things I didn't want to. Will fix.